impl/src/main/java/impl/UserResourceImpl.java

```diff
@@ -92,8 +94,21 @@ public UserResourceImpl(UserDao userDao, ResponseHeaderHolder responseHeaderHold
 
     private void addAsCookie(final ApplicationToken applicationToken, User user) {
         Map<String, Object> headers = new HashMap<>();
-        headers.put("Set-Cookie", "application=" + applicationToken.getApplicationToken() + "; path=/; domain=" + applicationTokenConfig.getDomain() + ";");
+        headers.put("Set-Cookie", formatCookie(applicationToken.getApplicationToken(), SESSION_MAX_AGE_SECONDS, applicationTokenConfig.getDomain()));
         responseHeaderHolder.addHeaders(user, headers);
     }
 
+    @Override
+    public Observable<Long> signOut(Auth auth) {
+        int expireNow = 0;
+        Map<String, Object> headers = new HashMap<>();
+        headers.put("Set-Cookie", formatCookie("deleted", expireNow, applicationTokenConfig.getDomain()));
```

Comment:
What is the point of writing "deleted" instead of clearing it? As far as I know, empty value and future expires is how to properly delete cookies.

Reply:
You cannot force all browsers to delete a cookie. The client can configure the browser in such a way that the cookie persists, even if it's expired. Setting the value as described above would solve this problem

Reply:
What I am asking is what is the point of setting "deleted" as value? Why not set it to empty?